pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -171,10 +176,11 @@ func (r *RepoSyncReconciler) Reconcile(ctx context.Context, req controllerruntim
 		return controllerruntime.Result{}, updateErr
 	}
 
+	// Create a tombstone, so we know to clean up managed objects during termination.
```

Comment:
This tombstone method won't survive rescheduling of the reconciler-manager pod. We might want to skip it and rely on the more durable method of just calling deleteManagedObjects again. 

Do we might need to verify that the RSync is managed by this reconciler-manager? The RSyncs don't have owner references and now neither do the other objects. Perhaps there's an annotation to identify the reconciler-manager?

Can multiple Config Syncs even co-exist on one cluster? Or do they end up fighting over every RSync? If not, we can remove the cache all together.